workshops/read_the_internet/README.md

```diff
@@ -0,0 +1,175 @@
+
+Spin up a Python3 repl by [clicking here](https://repl.it/languages/python3).
+
+Before we can start coding, we need to install a couple of external libraries. Follow the image below to find where to add the packages. Once in this menu you will need to install `beautifulsoup4`, `requests`, and `lxml`.
+
+![Repl.it packages menu](https://cloud-p5pdve0kf.vercel.app/0ezgif.com-video-to-gif.gif)
+
+#### What each package does
+`lxml` is a C package used by beautiful soup that parses the HTML, `beautifulsoup4` is for working with the parsed HTML, and `requests` is for requesting the webpage/html.
+
+<details>
+
+<summary> Learn more about the packages: </summary>
+
+- [BS4](https://www.crummy.com/software/BeautifulSoup/bs4/doc/)
+- [requests](https://requests.readthedocs.io/en/master/)
+- [LXML](https://lxml.de) 
+
+</details>
+
+## Lets start coding!
+
+![Start coding gif](https://cloud-5bmb4t5tp.vercel.app/1start-coding.gif)
+
+Once you've installed the packages, open the `main.py` file in the sidebar on the left. Then, import the packages you just installed.
+
+```python
+from bs4 import BeautifulSoup as bs
+import requests
+```
+
+Now we can start webscraping! We will begin by requesting our webpage and extracting the HTML from the request object.
+
+```python
+url = 'https://ncov2019.live/data'
+# request the website
+r = requests.get(url)
+# get the HTML from the request
+page = r.text
+```
+
+Next we will create the BeautifulSoup object and use `lxml` to parse the HTML.
+
+```python
+soup = bs(page, 'lxml')
+```
+
+Now that we have our BeautifulSoup object we can go all of the HTML! 
+
+### Working with the parsed HTML
+
+![HTML gif](https://cloud-5bmb4t5tp.vercel.app/2html.gif)
+
+In the next step, we'll be working with data from an HTML table. So, before, we continue, a quick mini-lesson on HTML tables:
+
+Here's an example of an HTML table:
+
+```html
+<table>
+    <tr>
+        <td>Country</td>
+        <td>Cases</td>
+        <td>Column</td>
+        <td>Column 2</td>
+    </tr>
+    <tr>
+        <td>United States</td>
+        <td>1,000,000</td>
+        <td>Column data</td>
+        <td>Column 2 data</td>
+    </tr>
+    <tr>
+        <td>Canada</td>
+        <td>100,000</td>
+        <td>Column data</td>
+        <td>Column 2 data</td>
+    </tr>
+</table>
+```
+
+A HTML table can be seen as a 2 dimensional array or list. The table contains rows (`tr` tags). Each row contains columns (`td` tags). Within the `td` tags is the data or text we can webscrape.
+
+### Python time!
+
+We're going to use BeautifulSoup to find an HTML table element. We want to find a specific table by ID `sortable_table_world` as seen in the picture. The code below finds the first table that matches that ID by using the `find` function and specifying the id as an attribute. The `attrs` is an optional argument and is a dictionary that can be set to any id, class, or HTML identifier.
```

Comment:
During my first few reads, it wasn't totally clear to me what exactly this meant. I kinda get it now, but I'm still having trouble picturing why we're getting data from an HTML table or why this is necessary for webscraping, and I'm having trouble picturing what this step is actually doing. I think it'll be a lot more confusing to club members who are doing this workshop.